frontend/www/js/omegaup/components/problem/StatementEdit.vue

```diff
@@ -153,9 +154,31 @@ export default class ProblemStatementEdit extends Vue {
   errors: string[] = [];
   languages = ['es', 'en', 'pt'];
   statements: types.Statements = {};
+  markdownConverter = markdown.markdownConverter({
+    preview: true,
+    imageMapping: {},
+  });
+  markdownEditor: Markdown.Editor = new Markdown.Editor(
+    this.markdownConverter,
+    '-statements',
+  );
+  markdownPreview: string = '';
 
   mounted(): void {
-    MathJax.Hub.Queue(['Typeset', MathJax.Hub, this.preview]);
+    const markdownConverter = markdown.markdownConverter({
+      preview: true,
+      imageMapping: {},
+    });
+    const preview = this.preview;
+    this.markdownEditor = new Markdown.Editor(
+      this.markdownConverter,
+      '-statements',
+    );
+
+    this.markdownEditor.hooks.chain('onPreviewRefresh', () => {
+      MathJax.Hub.Queue(['Typeset', MathJax.Hub, preview]);
```

Comment:
```suggestion
      MathJax.Hub.Queue(['Typeset', MathJax.Hub, this.preview]);
```

esto funciona? si sí, se puede eliminar L172?

Reply:
Listo, parece que si funcionó